src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+$square: 100px;
+$gap: 1px;
+$mainPadding: 10px;
+$days: "mon" 0,
+  "tue" ($square + $gap),
+  "wen" ($square + $gap) * 2,
+  "thu" ($square + $gap) * 3,
+  "fri" ($square + $gap) * 4,
+  "sat" ($square + $gap) * 5,
+  "sun" ($square + $gap) * 6;
+
+.page__calendar {
+  margin: 0;
+  width: $square * 7 + $gap * 6 + $mainPadding * 2;
+}
+
+.calendar {
```

Comment:
please add calendar block to `calendar.css` file and import it to `main.scss` with other files(`reset.scss`, `page.scss`)